### DISCUSSION TITLE
Something Awesome missing? Share your suggestions for the list here

### DISCUSSION THREAD

**Discussion by ricardofeynman** [ricardofeynman/Awesome-MC-707#2] [Ideas]
I have plenty more in mind to add, but will have to go digging through my bookmarks for the crème de la community. Anyone who hosts a GH repo dedicated to MC stuff can of course just do a PR if they prefer, and I'll get them added in.

**Comment by videopunks** [comment]
Looks like Splunge has some interesting .mpj tools here:
https://splunge.foo

Also - this is a CLI tool that lets you convert JD-08 SVD files into SVZ that _should__ be able to be read by the MC-707/MC-101, but I haven't tested it yet
https://github.com/sagamusix/JDTools
